Toolset/libraries/revideextensionlibrary.livecodescript

```diff
@@ -13,6 +13,52 @@ on extensionFinalize
    end if
 end extensionFinalize
 
+local sModuleVersion
+private command __EnsureModuleVersion
+   if sModuleVersion is not empty then
+      exit __EnsureModuleVersion
+   end if
+   local tTempFile, tModule, tFolder
+   put the tempname into tTempFile
+   set the itemdelimiter to slash
+   put item 1 to -2 of tTempFile into tFolder
+   put "module com.livecode.dummy" into tModule
+   put return & "end module" after tModule
+   put tModule into url("file:" & tTempFile)
+   revIDEExtensionCompile tTempFile, "", tFolder, "module.lcm"
+   put __CompiledModuleVersion(tFolder & slash & "module.lcm") \
+         into sModuleVersion
+end __EnsureModuleVersion
+
+private function __CompiledModuleVersion pModule
```

Comment:
@livecodeali should we have an engine function to get this do you think?

Reply:
Is that necessary when you can just parse the first few bytes of the .lcm file?

Reply:
There's a slight subtlety here I suppose with the fact that the output format of the lc-compile bundled with the IDE is not necessarily that of the engine, as they are separate programs - one of them might have been replaced. So it *might* make sense to also have that (essentially the VM version) as an engine function and make sure that they match before doing anything. OTOH maybe we shouldn't bother too much about the case that one or other program has been replaced!